docs/FLYSKY_FRM303.md

```diff
@@ -0,0 +1,112 @@
+# mLRS Documentation: FlySky FRM303 Transmitter Module #
+
+([back to main page](../README.md))
+
+The FlySky FRM303 transmitter module is a nearly perfect option for building mLRS equipment in the 2.4 GHz frequency range. Its main drawback is its price. It also has a small MCU which limits the available mLRS features. However, no hardware tweaking is required (although mods like the OLED mod are possible).
+
+<table>
+  <tbody>
+    <tr>
+      <td>Frequency Band</td>
+      <td>2.4 GHz</td>
+    </tr>
+    <tr>
+      <td>Max. RF Output Power</td>
+      <td>30 dBm (1 W) (the hardware is capable of 33 dBm/2 W, which is not enabled by default however)</td>
```

Comment:
Should a hint be provided on how to get a 33 dBm power output?  Something like 'a custom firmware build is required to make this option available'

Reply:
well, it's uncharted territory currently. But everyone is of course free to do whatthey want. Maybe one rather should say that it's currently not exploited by mLRS?

Reply:
suggestion for a text?

Reply:
How about:

30 dBm (1 W).  The hardware should be capable of 33 dBm (2 W) but current mLRS firmware does not provide this option.

Reply:
THANKS SO MUCH!